tools/nl/embeddings/utils_test.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Google LLC
```

Comment:
did you mean to include this test file?

Reply:
yes, otherwise need to remove this test in run_test.sh. I plan to add unittest in a following PR.